.github/workflows/convert.yml

```diff
@@ -0,0 +1,71 @@
+name: convert-yaml-to-json
+
+on:
+  push:
```

Comment:
Do we want this to be limited to specific branches to avoid over-commiting PRs?

Reply:
How about on push to main and on pull requests?

Reply:
@aaspinwall's suggestion sounds good to me!